resources/compile-config.sh

```diff
@@ -5,19 +6,18 @@ cache_dir="$HOME/Library/Caches/com.runningwithcrayons.Alfred-2/Workflow Data/co
 mkdir -p "$cache_dir"
 
 # Get paths to configuration file
-installed_config="./config.applescript"
-cached_config="$cache_dir/config.applescript"
+installed_config="config.applescript"
```

Comment:
I suggest we add a line with `installed_config_dir="."` and replace that line with `installed_config="$installed_config_dir/config.applescript"` instead.

Using `./` doesn't prevent errors in, as you still have to know that it references a file is in the cwd.

For example these two snippets are equivalent:
```
installed_config="./config.applescript"
# ...
echo "config_dir/installed_config"
```

```
installed_config="config.applescript"
# ...
echo "config_dir/$installed_config"
```